app/models/vm_or_template.rb

```diff
@@ -47,11 +47,19 @@ class VmOrTemplate < ApplicationRecord
     "unknown"     => "Unknown"
```

Comment:
Not all of these have equivalent provider plugins (e.g. parallels, xen) so we can't completely remove a small list in core

Reply:
Where do we even use those vendor types?

Reply:
They are set by the provider in refresh: https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb#L57

And shown on the UI https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/textual_mixins/textual_vmm_info.rb#L27-L42

Reply:
@agrare but do we support Xen and Parallels, in particular?  I highly doubt it.  So, let's get rid of them, if possible.